chapters/introduction.tex

```diff
@@ -30,94 +39,116 @@ \section{Doing credible research at scale}
+The \textbf{DIME Wiki} is one of our flagship products, a free online collection of our resources and best practices.\sidenote{
+\url{http://dimewiki.worldbank.org/}}
+
+This book complements the DIME Wiki by providing a structured narrative of the data workflow for a typical research project. 
 We will not give a lot of highly specific details in this text,
-but we will point you to where they can be found,
-and give you a sense of what you need to find next.
-Each chapter will focus on one task,
-and give a primarily narrative account of:
+but we will point you to where they can be found.\sidenote{Like this: 
+\url{https://dimewiki.worldbank.org/wiki/Primary_Data_Collection}}
+Each chapter focuses on one task, providing a primarily narrative account of:
 what you will be doing; where in the workflow this task falls;
-when it should be done; who you will be working with;
-why this task is important; and how to implement the most basic form of the task.
-
-We will use broad terminology throughout this book
-to refer to different team members:
+when it should be done; and how to implement it according to best practices.
+
+
+
+\section{Outline of this book}
+The book progresses through the typical workflow of an empirical research project.
+We start with ethical principles to guide empirical research, 
+focusing on research transparency and the right to privacy. 
+The second chapter discusses the importance of planning data work at the outset of the research project- 
+long before any data is acquired - and provide suggestions for collaborative workflows and tools. 
+Next, we turn to common research designs for \textbf{causal inference}{\sidenote{causal inference: identifying the change in outcome \textit{caused} by a particular intervention}}, and consider their implications for data structure. 
+The fourth chapter covers how to implement sampling and randomization to ensure research credibility, 
+and includes details on power calculation and randomization inference. 
+The fifth chapter provides guidance on high quality primary data collection, particularly for projects that use surveys. 
+The sixth chapter turns to data processing, 
+focusing on how to organize data work so that it is easy to code the desired analysis. 
+In the final chapter, we discuss publishing collaborative research- 
+both the research paper and the code and materials needed to recreate the results.  
+
+We will use broad terminology throughout this book to refer to research team members:
 \textbf{principal investigators (PIs)} who are responsible for
-the overall success of the project;
+the overall design and stewardship of the study;
 \textbf{field coordinators (FCs)} who are responsible for
-the operation of the project on the ground;
+the implementation of the study on the ground;
 and \textbf{research assistants (RAs)} who are responsible for
-handling technical capacity and analytical tasks.
+handling raw data processing and analytical tasks.
```

Comment:
```suggestion
handling data processing and analytical tasks.
```

I see what you mean by adding raw here, but I do not think it helps to specify that in this context